src/components/LabeledCheckbox/LabeledCheckbox.less

```diff
@@ -0,0 +1,41 @@
+.lucid-LabeledCheckbox {
+	align-items: center;
+	color: @color-darkGray;
+	cursor: pointer;
+	display: flex;
+	font: @font;
+	justify-content: flex-start;
+
+	&&-is-disabled {
+		color: @color-mediumGray;
+		cursor: not-allowed;
+	}
+
+	&:hover .lucid-Checkbox-visualization-container {
+		border-color: @color-primary;
+		transition: @Checkbox-animation-timing;
```

Comment:
It would put it into the global scope, but only if you load `Checkbox.less` before loading this file, otherwise it'll blow up.